packages/platform/platform-router/test/routers-injection.integration.spec.ts

```
@@ -34,9 +33,9 @@ describe("Routers injection", () => {
     const router = platformRouters.from(CustomStaticsCtrl);
     const router1 = platformRouters.from(CustomStaticsCtrl);
 
-    const provider = injector.getProvider<ControllerProvider>(CustomStaticsCtrl)!;
-    const router2 = injector.get(provider.tokenRouter);
-    const controller = injector.invoke<CustomStaticsCtrl>(CustomStaticsCtrl)!;
+    const provider = injector().getProvider<ControllerProvider>(CustomStaticsCtrl)!;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)